### DISCUSSION TITLE
Rename

### DISCUSSION THREAD

**Discussion by mitza-oci** [oci-labs/check-ins#1568] [Ideas]
Check-Ins is a specific feature / user story.  Name the repository (and the deployed site) something else.

**Comment by mkimberlin** [comment]
I'm not against having a broader name...I'd rather not do that willy-nilly though. I'd love to hear some brainstorming on names that would make since. The idea behind the Check-Ins name (I regret the 's') is that it flashes to the idea of a place to check-in, as well as to the Professional Development process itself.  This project's general goal is to be a fully-featured, user-friendly team engagement tool.  Anyone have name suggestions that fit those ambitions?

**Reply by mkimberlin** [reply]
I want to revive this conversation...Perhaps I should toss this into the internal chat for discussion.
